.github/workflows/main.yml

```diff
@@ -17,11 +17,11 @@ jobs:
       - uses: actions/checkout@v4
       - uses: chartboost/ruff-action@v1
         with:
-          src: './src'
+          src: './src/fcp'
```

Comment:
Why not the full src path?

Reply:
My bad, was trying to run the CI locally and forgot to unstage the change